src/Logger/Logger.cc

```diff
@@ -78,3 +84,25 @@ void InitLogger(bool no_log_file, int verbosity) {
         spdlog::info("Writing to the log file: {}", log_fullname);
     }
 }
+
+void LogReceivedEventType(const CARTA::EventType& event_type) {
+    auto event_name = CARTA::EventType_Name(CARTA::EventType(event_type));
+    if (!event_name.empty()) {
+        if (log_icd_messages) {
+            spdlog::debug("<== {}", event_name);
+        }
+    } else {
+        spdlog::error("Unknown event type: {}!", event_type);
+    }
```

Comment:
Please disregard my previous remark about logging this even when we aren't logging ICD messages -- we already log an error for an unrecognised received message in `OnMessageTask`. And I don't think it makes sense to check for unrecognised outgoing messages. So we can put the `log_icd_messages` check at the top level, and also log these unknown event type messages to `debug` like the other protocol messages.